fearless/Modules/Wallet/Contacts/ContactViewModel.swift

```diff
@@ -37,55 +37,7 @@ extension ContactViewModel: WalletCommandProtocol {
     var command: WalletCommandProtocol? { self }
 
     func execute() throws {
```

Comment:
As we discussed it is better to pass a command instead of delegate proxy. So you don't need to store delegate proxy as a field in the factory

Reply:
The problem is that command is a get-only property and it returns viewModel. I don't quite understand how to change it to make it do phishing check in one case and to keep its behaviour in all the other cases. Let's discuss